File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Models/ConferenceOwner.cs
using System;

namespace ConferenceManagement
{
    public class ConferenceOwner
    {
        public string Name { get; private set; }
        public string Email { get; private set; }

        public ConferenceOwner() { }
        public ConferenceOwner(string name, string email)
        {
            Name = name;
            Email = email;
        }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Models/ConferenceOwner.java
package com.microsoft.conference.management.domain.Models;

public class ConferenceOwner {
    public String Name;
    public String Email;

    public ConferenceOwner() {
    }

    public ConferenceOwner(String name, String email) {
        Name = name;
        Email = email;
    }
}
